profile/README.md

```diff
@@ -3,77 +3,59 @@
+# Code of Conduct
+Participation in the InstructLab community is governed by our [Code of Conduct](https://github.com/instructlab/community/blob/main/CODE_OF_CONDUCT.md)
 
-* [Repo](https://github.com/instructlab/taxonomy)
-* [Contributing](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md)
+# Quick Links
 
-### [Command-line Interface](https://github.com/instructlab/instructlab)
+## Governance
 
-This command-line interface for InstructLab will allow you to create models tuned with data you provide using the InstructLab method on your laptop or workstation.
+Overview of how InstructLab project [governance](https://github.com/instructlab/community/blob/main/governance.md) operates.
 
-* [Repo](https://github.com/instructlab/instructlab)
-* [Contributing](https://github.com/instructlab/instructlab/blob/main/CONTRIBUTING/CONTRIBUTING.md)
+## Security
 
-> [!IMPORTANT]
-> It is important to understand that running InstructLab on a laptop will give you a low-fidelity approximation of both synthetic data generation (using the `ilab generate` command)
-> and model instruction tuning (using the `ilab train` command, which uses QLoRA.) The quality of the results you get using these tools on a laptop will not be as high-fidelity as they might be using
-> a larger teacher model and a different training method. We have optimized InstructLab to enable community members with modest hardware to be able to use the technique. If you have more sophisticated
-> hardware, you can configure InstructLab to use a larger teacher model [such as Mixtral](https://huggingface.co/docs/transformers/model_doc/mixtral) in order to achieve higher-fidelity results.
-  
-## Model Training Infrastructure
-
-The infrastructure used to regularly train the model based on new contributions from the community is GPU intensive and generously donated and maintained by IBM.
+Security policies and practices, including reporting vulnerabilities can be found in our [Security.md](https://github.com/instructlab/community/blob/main/SECURITY.md).
 
----
+## Read the Paper
 
-<!--
-## Additonal aspects of the project include:
+InstructLab 🐶 uses a novel synthetic data-based alignment tuning method for Large Language Models (LLMs.) The "lab" in InstructLab 🥼 stands for [**L**arge-Scale **A**lignment for Chat**B**ots](https://arxiv.org/abs/2403.01081) [1].
 
-- [community](): community content
-- [github-bots](): automated CI/CD bots and related content
--->
+[1] Shivchander Sudalairaj*, Abhishek Bhandwaldar*, Aldo Pareja*, Kai Xu, David D. Cox, Akash Srivastava*. "LAB: Large-Scale Alignment for ChatBots", arXiv preprint arXiv: 2403.01081, 2024. (* denotes equal contributions)
 
-### Code of Conduct & Covenant
+### Acknowledgements
```

Comment:
Dropped back a level because it doesn't seem like a sub-section of "Quick Links"

```suggestion
## Acknowledgements
```